### ISSUE TITLE
pry-byebug causes blocking of test rails server

### ISSUE THREAD

**Issue by joeletizia** [deivid-rodriguez/pry-byebug#84]
[Repo that reproduces issue](https://github.com/joeletizia/pry-byebug-issue)

## Summary
When pry-byebug is added to this project's gem file and `binding.pry` is used within a feature test (RSpec or Cucumber with Capybara) the web thread/process waits on the `binding.pry` even if that breakpoint is located in the test file. When pry-byebug is removed, the web thread/process is not blocked and the user may click around around if they are using a non-headless browser (Selenium w/ Firefox in this case). 

### Steps to Reproduce
1. Clone this repo.
2. `bundle`
3. `rake db:create`
4. `rspec spec/features/users/sign_in_spec.rb:11`
5. When the break point is reached, attempt to click a link to a new page.

#### Expected Result
The server is able to handle the request and renders the page.

#### Actual Result
The server hangs waiting on the binding.pry call.

6. Continue on from the breakpoint. 
7. Remove `gem 'pry-byebug'` from the gem file.
8. `rspec spec/features/users/sign_in_spec.rb:11`
9. When the break point is reached, attempt to click a link to a new page.

#### Actual Result
The server does not wait on the binding.pry call.

Is this intended behavior? Maybe this is related to #69?

**Comment by deivid-rodriguez** [comment]
Hi @joeletizia, thanks for the report. I think this is a duplicate of #69 indeed, so I'm going to close this. Feel free to reopen if you disagree.

**Comment by deivid-rodriguez** [comment]
Never actually closed it...
